src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -0,0 +1,444 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=inconsistent-return-statements
+# pylint: disable=unused-variable
+
+import re
+
+
+# Namespace Region
+def cli_namespace_create(client, resource_group_name, namespace_name, location=None, tags=None, sku='Standard',
+                         capacity=None):
+    from azure.mgmt.servicebus.models import SBNamespace, SBSku
+    return client.create_or_update(
+        resource_group_name=resource_group_name,
+        namespace_name=namespace_name,
+        parameters=SBNamespace(
+            location,
+            tags,
+            sku=SBSku(
+                sku,
+                sku,
+                capacity)
+        )
+    )
+
+
+def cli_namespace_update(instance, tags=None, sku='Standard', capacity=None):
+
+    if tags is not None:
+        instance.tags = tags
+
+    if sku is not None:
+        instance.sku.name = sku
+        instance.sku.tier = sku
+
+    if capacity is not None:
+        instance.sku.capacity = capacity
+
+    return instance
+
+
+def cli_namespace_list(client, resource_group_name=None):
+    if resource_group_name:
+        return client.list_by_resource_group(resource_group_name=resource_group_name)
+
+    if not resource_group_name:
+        return client.list()
```

Comment:
`if not resource_group_name:` is redundant. Just return `client.list()`

Reply:
implemented